src/main/java/edu/ucsb/cs156/happiercows/controllers/AnnouncementsController.java

```diff
@@ -133,54 +142,70 @@ public ResponseEntity<Object> getAnnouncementById(@Parameter(description = "The
     @PreAuthorize("hasAnyRole('ROLE_USER', 'ROLE_ADMIN')")
```

Comment:
I think it was actually a mistake before to let "any user" update announcements.  That really should be reserved only for Admins, especially since at the moment, until we restrict game via the course roster, literally anyone with a Google account could log in and be a user.   The potential for bad outcomes (i.e. someone sabotaging the game) is high.
```suggestion
    @PreAuthorize("hasRole( 'ROLE_ADMIN')")
```

Plus this simplifies your code later on.

Reply:
This makes sense to me, I'll change the code